pmultiqc/modules/common/dia_utils.py

```
@@ -0,0 +1,719 @@
+    df["is_contaminant"] = df["Protein.Names"].str.startswith("CON", na=False)
+
+    # 3. "Charge"
+    charge = dict()
+    for raw_file, group in df[["Run", "Precursor.Charge"]].groupby("Run"):
+        charge[raw_file] = group["Precursor.Charge"].value_counts()[2] / len(group)
+    charge_median = np.median(list(charge.values()))
+    heatmap_charge = dict(
+        zip(
+            charge.keys(),
+            list(map(lambda v: float(1 - np.abs(v - charge_median)), charge.values())),
+        )
```

Ruff (0.14.0)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)